twingate/resource_resource_test.go

```diff
@@ -24,6 +26,15 @@ const (
 	udpPorts       = "protocols.0.udp.0.ports.#"
 )
 
+func wait() resource.TestCheckFunc {
```

Comment:
@vmanilo whats the best place to put this function?  I have a feeling we'll need it on other ACC tests too...

Reply:
maybe let's add to latest tests that were failing, I will do it

Reply:
@vmanilo no I mean what file should the `wait` function itself be on...

Reply:
I think we can keep it in this file

Reply:
it will be accessible in other acc tests, as it's the same package